languages/rust/oso/tests/common/mod.rs

```diff
@@ -74,7 +74,7 @@ impl OsoTest {
 
     pub fn qvar_one<T>(&mut self, q: &str, var: &str, expected: T)
     where
-        T: oso::FromPolar + PartialEq<T> + std::fmt::Debug,
```

Comment:
Why did this change to `FromPolarValue` instead of `FromPolar`?

Reply:
So I could write the tests with PolarValue instead of Value